counter/src/test/java/org/infinispan/counter/StrongCounterTest.java

```diff
@@ -96,6 +96,58 @@ public void testCompareAndSet(Method method) {
       }
    }
 
+   public void testCompareAndSetConcurrent(Method method) throws ExecutionException, InterruptedException, TimeoutException {
+      final int numThreadsPerNode = 2;
```

Comment:
Increasing this to 10 currently leads to https://issues.jboss.org/browse/ISPN-8134